lists/et.csv

```diff
@@ -177,6 +177,10 @@ https://www.hrw.org,HUMR,Human rights,2014-04-15,citizenlab,
 https://www.privacyinternational.org,HUMR,Human rights,2014-04-15,citizenlab,
 https://www.torproject.org,FEXP,Free expression and media freedom,2014-04-15,citizenlab,
 https://www.twitter.com,BLGSERV,Blogging domains and blogging services,2014-04-15,citizenlab,
-https://www.oromiamedia.org/,NEWS,News Media,2016-05-30,OONI,
-https://www.facebook.com/Jawarmd,NEWS,News Media,2016-05-30,OONI,
-https://www.dropbox.com/s/n65b3d67f82asn2/Leaked%20National%20Entrance%20Exam_English.pdf?dl=0,FILE,File-sharing,2016-05-30,OONI,
+https://www.oromiamedia.org/,NEWS,News Media,2016-05-30,OONI,
+https://www.facebook.com/Jawarmd,NEWS,News Media,2016-05-30,OONI,
+https://www.dropbox.com/s/n65b3d67f82asn2/Leaked%20National%20Entrance%20Exam_English.pdf?dl=0,FILE,File-sharing,2016-05-30,OONI,
+http://www.oromiamedia.org/,NEWS, News Media, 2016-09-09, CIPIT
+http://www.zehabesha.com/ ,NEWS, News Media, 2016-09-09, CIPIT
+http://www.mereja.com/,NEWS, News Media, 2016-09-09, CIPIT
+https://www.mereja.com/,NEWS, News Media, 2016-09-09, CIPIT

```

Comment:
Can you fix the extra whitespace from line 183-186 and ensure that there are always 6 columns (a missing `,` at the end).